package.json

```diff
@@ -95,9 +93,8 @@
     "start": "yarn start:pf3",
     "start:cypress": "lerna run cypress:open",
     "start:demo-app": "lerna run start:demo-app --stream",
-    "start:pf3": "concurrently \"yarn storybook\" \"yarn storybook:openurl\"",
+    "start:pf3": "yarn storybook",
     "start:pf4": "yarn build:pf4 && lerna run develop --parallel --stream",
-    "storybook:openurl": "node ./storybook/openBrowser.js",
```

Comment:
Is this not needed anymore?

Reply:
Nope, the new storybook opens by itself and prints a message to the console with the link.